### DISCUSSION TITLE
2023년 03월 16일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#10] [General]
# 2023년 03월 16일

LOG 체스 2단계 PR 보냈다. 너무 질문할 내용이 없어서 그런지 리뷰가 5개밖에 달리지 않았다 ㅋㅋㅋㅋ 좀 더 많았으면 좋았겠지만 이건 이것대로 괜찮다고 생각한다. 체인저랑 나랑 주말에 생일이어서 온보딩 조원들이 아이스크림 케익 사와서 축하해줬다…

https://cyma-s.github.io/2023-03-16/

**Comment by Choi-JJunho** [comment]
나는 코코닥꺼만 못봤는데..
베로 지식교환하자
